.gitlab-ci.yml

```diff
@@ -335,12 +336,12 @@ New OSTree:
   parallel:
     matrix:
       - RUNNER:
-          - rhos-01/fedora-35-x86_64-large
-          # - rhos-01/fedora-36-x86_64-large
-          - rhos-01/rhel-8.7-nightly-x86_64-large
-          - rhos-01/rhel-9.1-nightly-x86_64-large
-          - rhos-01/centos-stream-8-x86_64-large
-          - rhos-01/centos-stream-9-x86_64-large
+          - rhos-01/fedora-35-x86_64
+          # - rhos-01/fedora-36-x86_64
```

Comment:
ATM there are fedora-36-x86_64 and fedora-36-x86_64-large runner definitions so you can uncomment this line.